api/src/controllers/connected-user-log.js

```diff
@@ -0,0 +1,42 @@
+const auth = require('../auth');
+const serverUtils = require('../server-utils');
+const connectedUserLogService = require('../services/connected-user-log');
+
+module.exports = {
+  log: (req) => {
+    return auth
+      .getUserCtx(req)
+      .then((userCtx) => {
+        const log = {
+          user: userCtx.name,
+          timestamp: new Date().getTime()
+        };
+        connectedUserLogService.save(log);
+      });
+  },
+  get: (req, res) => {
+    return auth
+      .getUserCtx(req)
+      .then((userCtx) => {
+        if (!auth.isDbAdmin(userCtx)) {
+          throw {
+            code: 401,
+            message: 'User is not an admin'
+          };
+        }
```

Comment:
I'm tempted to ask that this be moved to a middleware to handle permissions. I don't see a clear established pattern that this might follow though. I may be missing something though. Please do at least give it a look to see where permissions logic might be separated (and then tested at the routing level) from the logic of getting log data.